draft-regext-rdap-extensions.md

```diff
@@ -54,144 +54,236 @@ Domain Name Registries (DNRs), Regional Internet Registries
 RDAP queries are defined in [@!RFC9082] and RDAP responses are defined
 in [@!RFC9083].
 
-RDAP contains a means to define extensions for queries not found in [@!RFC9082] and
-responses not found in [@!RFC9083]. RDAP extensions are also described in [@!RFC7480].
-This document uniformly describes RDAP extensions, clarifies their usage, and
-defines additional semantics that were previously undefined or ambiguous.
+RDAP contains a means to define extensions for queries not found in
+[@!RFC9082] and responses not found in [@!RFC9083]. RDAP extensions
+are also described in [@!RFC7480].  This document describes the
+requirements for RDAP extension definition and use, clarifying
+ambiguities and defining additional semantics and options that were
+previously implicit.
 
 ## Summary of Updates
 
 This document updates [@!RFC7480], [@!RFC9082], and [@!RFC9083] in the following
 areas:
 
-1. (#extension_identifier) provides additional guidance on RDAP extension identifiers,
-   registration of identifiers for the purpose of avoiding namespace collisions,
-   and reserves a specific syntax for future use.
-1. (#usage_in_queries) clarifies the usage of extension identifiers in RDAP URLs and formally defines
+1. (#purpose) provides additional guidance on the purpose
+   of RDAP extension identifiers, including their registration for the
+   purpose of avoiding namespace collisions, as well as for signaling
+   server policy/behavior.
+1. (#syntax) reserves a specific identifier syntax for future use.
+1. (#usage_in_requests) clarifies the usage of extension identifiers in RDAP URLs and formally defines
    their usage with child path segments and query parameters.
-1. (#usage_in_json) clarifies the usage of extension identifiers in JSON.
-1. (#object_classes_in_extensions) and (#search_results_in_extensions) disambiguates the definition
-   of new object classes and search results by extensions in [@!RFC9082]. 
-1. This document describes the various styles and kinds of RDAP extensions
-   and their extension identifiers.
-1. This document describes extension versioning and changes to extensions
-   that would make them incompatible or compatible with previous versions of an extension.
-1. (#redirects) and (#referrals) discusses the uses of extensions with regard
-   to RDAP redirects and referrals, including security and privacy considerations.
-1. This document provides guidance to extension authors regarding the creation of extension specification.
+1. (#usage_in_responses) clarifies the usage of extension identifiers
+   in responses, including for new object classes and search result arrays.
+1. (#extension_implementer_considerations) documents behavior
+   considerations for extension implementers.
+1. (#extension_author_considerations) documents behavior
+   considerations for extension developers, covering
```

Comment:
developers -> authors since "developers" might get confused with "implementors" and that section's title already includes the word "author".

Reply:
Thanks, updated.